source/controllers/LegacyStepsController.mc

```diff
@@ -2,22 +2,46 @@ import Toybox.Application;
 import Toybox.Complications;
 import Toybox.Lang;
 
-class LegacyStepsController {
-  function getLabel() as String {
+class LegacyStepsController extends BaseController {
+  function initialize() {
+    BaseController.initialize();
+  }
+
+  public function getAngle() as Numeric {
```

Comment:
We still need to figure out a nicer way of indicating when the user is already past the steps goal